examples/client-secret-basic/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+
+server.port=3001
```

Comment:
Can we get this from the config? Or, do we need the config if this is set in this other file?

Reply:
inverstigating